modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -28,13 +30,17 @@ dependencies {
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "2.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "2.0.0"
 #elseif (${liferayVersion.startsWith("7.1")})
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib", version: "3.2.2"
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib.util.freemarker.contributor", version: "2.0.0"
```

Comment:
what is this dependency used for?

Reply:
I'm not 100% sure. I added these taglibs based on keywords. Freemarker and util haha